components/UsersList/UsersList.tsx

```diff
@@ -24,15 +25,24 @@ export function UsersList({
   orgGuid: string;
 }) {
   const [removedUserGuids, setRemovedUserGuids] = useState([] as string[]);
+  const [searchValue, setSearchValue] = useState('' as string);
 
   function usersSorted(usersList: Array<UserObj>): Array<UserObj> {
     return sortObjectsByParam(usersList, 'username');
   }
 
   function usersFiltered(usersList: Array<UserObj>): Array<UserObj> {
```

Comment:
[non-blocking question] Can you remind me why we had any user filtering before this PR?  Is it when you remove a user from the org?

Reply:
yep, it was for removing the user without needing to do another API request.